fe/fe-core/src/main/java/org/apache/doris/planner/EsScanNode.java

```diff
@@ -250,7 +251,10 @@ private List<TScanRangeLocations> getShardLocations() throws UserException {
                 TEsScanRange esScanRange = new TEsScanRange();
                 esScanRange.setEsHosts(shardAllocations);
                 esScanRange.setIndex(shardRouting.get(0).getIndexName());
-                esScanRange.setType(table.getMappingType());
+                // 8.x type is cancelled
+                if (table.majorVersion == null || table.majorVersion.before(EsMajorVersion.V_8_X)) {
```

Comment:
Don't rely on this, don't issue this parameter directly, BE will do the empty judgment processing